2-mandatory/3-shopping-list.js

```diff
@@ -19,6 +19,17 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
```

Comment:
This implementation looks good to me!
For an extra challenge - can you re-write this using the `filter` array method?